app/lib/routeHandler.js

```diff
@@ -4,6 +4,63 @@ const fs = require('fs');
 const path = require('path');
 const expandPath = require('../../lib/expandPath');
 
+/**
+ *
+ * @param {*} res
+ * @param {*} next
+ * @param {*} promise
+ * @returns {bool} Whether or not the argument was a promise and was handled.
+ */
+const handlePromise = (res, next, promise) => {
+  if (!(promise instanceof Promise)) return false;
+
+  promise
+    .then(result => {
+      res.send(result);
+    })
+    .catch(err => next(err));
+
+  return true;
+};
+
+/**
+ *
+ * @param {*} res
+ * @param {*} next
+ * @param {*} promise
+ * @returns {bool} Whether or not the argument was a function and was handled.
+ */
+const handleFunction = (req, res, next, data) => {
+  if (typeof data !== 'function') return false;
+
+  try {
+    const result = data(req);
```

Comment:
This guy is rogue.... `data(req)` is executed on line 39 too.

Reply:
@ryanricard Fixed.